Formula/log4cxx.rb

```diff
@@ -14,24 +14,21 @@ class Log4cxx < Formula
     sha256 cellar: :any, high_sierra:   "11478b4f5ece24ec391954cc0538bb28f11ae6256a9499ca1e95103c2eb1d75c"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
   depends_on "libtool" => :build
 
   depends_on "apr-util"
 
   def install
-    # Fixes build error with clang, old libtool scripts. cf. #12127
-    # Reported upstream here: https://issues.apache.org/jira/browse/LOGCXX-396
-    # Remove at: unknown, waiting for developer comments.
-    system "./autogen.sh"
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          # Docs won't install on macOS
-                          "--disable-doxygen",
-                          "--with-apr=#{Formula["apr"].opt_bin}",
-                          "--with-apr-util=#{Formula["apr-util"].opt_bin}"
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..",
+                      *std_cmake_args,
+                      "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                      "-DCMAKE_CXX_STANDARD=17",
```

Comment:
Why is this needed?

Reply:
It is recommended to build with cxx17
https://github.com/apache/logging-log4cxx/blob/master/CMakeLists.txt#L38-L42

Reply:
Let's leave it out, and let the upstream `CMakeLists.txt` handle it for us. They'll set it to the correct value as long as we don't set it.